### PR TITLE
chore: updating packages and deprecating wonderlands isolmate

### PR DESCRIPTION
**Pull request by wei3erHase**
Closes BES-330

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,17 +33,17 @@
...
+    "@commitlint/config-conventional": "19.2.2",
+    "@defi-wonderland/natspec-smells": "1.1.1",
     "ds-test": "github:dapphub/ds-test#e282159",
-    "forge-std": "github:foundry-rs/forge-std#4513bc2",
+    "forge-std": "github:foundry-rs/forge-std#bb4ceea",
     "husky": ">=8",
     "lint-staged": ">=10",
-    "solhint": "github:solhint-community/solhint-community#v4.0.0-rc01",
-    "sort-package-json": "1.53.1"
+    "solhint": "github:solhint-community/solhint-community#v4.0.0",
```
Comment (gas1cent) [inline package.json:46]: Now that there is a released version, could you update the source to install it from NPM?
https://www.npmjs.com/package/solhint-community

Comment (wei3erHase) [reply package.json:46]: sure!

File: package.json [hunk 2]
```diff
@@ -33,17 +33,17 @@
...
+    "solmate": "github:transmissions11/solmate#a9e3ea2"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.0.3",
-    "@commitlint/config-conventional": "17.0.3",
-    "@defi-wonderland/natspec-smells": "1.0.3",
+    "@commitlint/cli": "19.3.0",
+    "@commitlint/config-conventional": "19.2.2",
+    "@defi-wonderland/natspec-smells": "1.1.1",
     "ds-test": "github:dapphub/ds-test#e282159",
```
Comment (0xOneTony) [inline package.json:42]: I think we should remove `ds-test` completely, isn't it part of forge-std ?

**Review by 0xOneTony — DISMISSED** [review]
Minor comment, lgtm other than that

File: package.json (hunk 2 above)
Comment (wei3erHase) [reply package.json:42]: if we remove it then `forge-std` fails to compile ⚰️

**Review by 0xOneTony — DISMISSED** [review]

**Review by 0xDiscotech — DISMISSED** [review]

**Review by 0xRaccoon — DISMISSED** [review]

**Review by 0xOneTony — APPROVED** [review]
LGTM!

**Review by 0xOneTony — APPROVED** [review]
nice!

**Review by 0xRaccoon — APPROVED** [review]

**Pull request merged**